docs/tutorials/custom_experiment.rst

```diff
@@ -0,0 +1,559 @@
+          
+          result = AnalysisResultData("counts", combined_counts)
+          return [result], []
+
+      @classmethod
+      def _swap_bitstring(cls, bitstring, sig):
+          """Swap a bitstring based signature to flip bits at."""
+          # This is very inefficient but demonstrates the basic idea
+          # Really should do with bitwise operations of integer counts rep
+          return "".join(reversed(
+              [cls._swap_bit[b] if sig[- 1 - i] else b for i, b in enumerate(bitstring)]
+          ))
+
+
+To test our code, we first simulate a noisy backend with asymmetric readout error in Aer:
+
+.. jupyter-execute::
+
+  from qiskit.providers.aer import AerSimulator, noise
+
+  backend_ideal = AerSimulator()
+
+  # Backend with asymetric readout error
+  p0g1 = 0.3
+  p1g0 = 0.05
+  noise_model = noise.NoiseModel()
+  noise_model.add_all_qubit_readout_error([[1 - p1g0, p1g0], [p0g1, 1 - p0g1]])
+  noise_backend = AerSimulator(noise_model=noise_model)
+
+Let's use a GHZ circuit as the input.
+
+.. jupyter-execute::
+
+    # GHZ Circuit
+    nq = 4
+    qc = QuantumCircuit(nq)
+    qc.h(0)
+    for i in range(1, nq):
+        qc.cx(i-1, i)
+    
+    qc.draw("mpl")
+
+Check that the experiment is appending a random Pauli and measurements as expected:
+
+.. jupyter-execute::
+
+    # Experiment parameters
+    total_shots = 100000
+    num_samples = 50
+    shots = total_shots // num_samples
+
+    # Run ideal randomized meas experiment
+    exp = RandomizedMeasurement(qc, num_samples=num_samples)
+    exp.circuits()[0].draw("mpl")
+
+We now run the experiment with a GHZ circuit on an ideal backend:
+
+.. jupyter-execute::
+
+    expdata_ideal = exp.run(AerSimulator(), shots=shots)
+    counts_ideal = expdata_ideal.analysis_results("counts").value
+    print(counts_ideal)
+
+Repeat the experiment on the backend with readout error and compare with results
+from running GHZ circuit itself:
+
+.. jupyter-execute::
+
+    # Run noisy randomized meas experiment with readout error
+    expdata_noise = exp.run(noise_backend, shots=shots)
+    counts_noise = expdata_noise.analysis_results("counts").value
+
+    # Run noisy simulation of the original circuit without randomization
+    meas_circ = qc.copy()
+    meas_circ.measure_all()
+    result = noise_backend.run(meas_circ, shots=total_shots).result()
+    counts_direct = result.get_counts(0)
+
+    from qiskit.visualization import plot_histogram
+
+    # Plot counts, ideally randomized one should be more symmetric in noise
+    # than direct one with asymmetric readout error
+    plot_histogram([counts_ideal, counts_direct, counts_noise],
+                legend=["Ideal",
+                        "Asymmetric meas error (Direct)",
+                        "Asymmetric meas error (Randomized)"])
+
+The asymmetric measurement of the original circuit on this backend (Direct on the plot legend)
+has been successfully symmetrized by the application of randomized measurement 
```

Comment:
Maybe you want a comment saying that for the GHZ state symmetrizing the error should produce a symmetric graph? I don't think that is true for an arbitrary input circuit.

Reply:
Added a sentence.